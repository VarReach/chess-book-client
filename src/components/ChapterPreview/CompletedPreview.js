import React from 'react';
import './ChapterPreview.css';

function CompletedPreview(props){
  return (
    <>
      <div className="chapter-preview__completion-indicator">
        <span>completed</span>
      </div>
      <div className="chapter-preview__completed-date">
        <span >
          {props.date}
        </span>
      </div>
    </>
  )
}

export default CompletedPreview;