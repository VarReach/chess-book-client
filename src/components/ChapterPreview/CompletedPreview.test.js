import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CompletedPreview from './CompletedPreview';

describe(`NotefulForm component`, () => {
  const props = {
    date: 'this is a date'
  };

  it('renders a CompletedPreview by default', () => {
    const wrapper = shallow(<CompletedPreview />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the CompletedPreview given props', () => {
    const wrapper = shallow(<CompletedPreview {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})