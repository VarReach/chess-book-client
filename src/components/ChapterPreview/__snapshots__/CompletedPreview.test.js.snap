// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotefulForm component renders a CompletedPreview by default 1`] = `
<Fragment>
  <div
    className="chapter-preview__completion-indicator"
  >
    <span>
      completed
    </span>
  </div>
  <div
    className="chapter-preview__completed-date"
  >
    <span />
  </div>
</Fragment>
`;

exports[`NotefulForm component renders the CompletedPreview given props 1`] = `
<Fragment>
  <div
    className="chapter-preview__completion-indicator"
  >
    <span>
      completed
    </span>
  </div>
  <div
    className="chapter-preview__completed-date"
  >
    <span>
      this is a date
    </span>
  </div>
</Fragment>
`;
